tripal_chado/config/install/tripal.tripalentitytype_collection.general_chado.yml

```
@@ -43,9 +43,9 @@ content_types:
         synonyms:
             - bio_data_4
 
-    -   label: Contact
-        term: local:contact
-        help_text: Use the contage page a person or institution that can be linked as a responsible party for data or results.
+    -   label: Communication Contact
+        term: NCIT:C47954
+        help_text: A channel for communication between groups.
```

For consistency we need to revert this change, but also fix the typos
```suggestion
        help_text: Use the contact page for a person or institution that can be linked as a responsible party for data or results.
```